.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = cuda
 	url = https://github.com/system76/cuda
 	branch = master
+[submodule "natron"]
+	path = natron
+	url = git@github.com:pop-os/packaging-natron.git
```

Comment:
I do not recommend using SSH - it requires the machine running the build to be authenticated.